src/main/test/base/NavigationTest.java

```diff
@@ -0,0 +1,87 @@
+package base;
+
+import org.junit.jupiter.api.Test;
+
+import base.constants.MapConstants;
+
+import base.navigationservice.NavigationService;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NavigationTest {
```

Comment:
It is good practice to name test class the same as real class, but with `Test` postfix. In this case it should be NavigationServiceClass.

Reply:
so NavigationServiceClassTest?

Reply:
should be `NavigationServiceTest`